graphql_compiler/schema_transformation/split_query.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from collections import namedtuple
+
+from graphql.language import ast as ast_types
```

Comment:
Wow you wrote this renamed import in a lot of places, huh :)

Reply:
Yeah... I can go change all of these at some point. There are possibly merged code with this kind of import too.